openvdb_points/tools/PointDataGrid.h

```diff
@@ -299,6 +299,22 @@ class PointDataLeafNode : public tree::LeafNode<T, Log2Dim> {
     /// @brief Returns whether an attribute exists. This method is faster
     /// than hasAttribute(const Name&) as it avoids a map lookup.
     /// @param pos    Index of the attribute
+    bool hasAttribute(const size_t pos) const
+    {
+        return pos < mAttributeSet->size();
+    }
+
+    /// @brief Returns whether an attribute exists.
+    /// @param attributeName    Name of the attribute
+    bool hasAttribute(const Name& attributeName) const
```

Comment:
I like this change, think we should also change the original methods to hasTypedAttribute() to distinguish them.